docs/asset-store/08-03-gateway-s3-mode-set-up.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Set Minio to the AWS S3 Gateway mode
+type: Tutorials
+---
+
+By default, you install Kyma with the Asset Store in Minio stand-alone mode. This tutorial shows how to set Minio to the AWS S3 Gateway mode using an [override](/root/kyma/#configuration-helm-overrides-for-kyma-installation).
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Amazon Web Services (AWS)](https://aws.amazon.com) account
+
+## Steps
+
+You can set Minio to the AWS S3 Gateway mode both during and after Kyma installation. In both cases, you need to create and configure a access key for AWS account, apply a Secret and Config Map with an override onto a cluster or Minikube, and trigger the Kyma installation process. 
+
+>**CAUTION:** Buckets created in Minio without using Bucket CRs are not recreated or migrated while switching to the Minio Gateway mode.
+
+>**CAUTION:** AWS S3 Gateway Mode is testes only manually.
```

Comment:
Please move those cautions to top of each doc
and write openly that this instruction was tested manually against kyma 1.6 while others are regularly tested on pipelines. 


maybe we should link here to future ticket?

Reply:
@derberg take a look if it is OK now. Yes I can link, but first I need to create it 😃